docs/pages/inner_workings.mdx

```diff
@@ -2,7 +2,7 @@ When calling `open-next build`, OpenNext **runs `next build`** to build the Next
 
 #### Building the Next.js app
 
-OpenNext runs the `build` script in your `package.json` file. Depending on the lock file found in the app, the corresponding packager manager will be used. Either `npm run build`, `yarn build`, or `pnpm build` will be run. For more on customizing the build command, see [overriding the build command](/advanced/options#custom-build-command).
+OpenNext runs the `build` script in your `package.json` file. Depending on the lock file found in the app, the corresponding packager manager will be used. Either `npm run build`, `yarn build`, or `pnpm build` will be run. For more on customizing the build command, see [overriding the build command](/config#build-arguments).
```

Comment:
This is not the correct link, to override the build command you have to use the `buildCommand` fields in the `open-next.config.ts` https://open-next.js.org/config/full_example

Reply:
> This is not the correct link, to override the build command you have to use the `buildCommand` fields in the `open-next.config.ts` https://open-next.js.org/config/full_example

![image](https://github.com/user-attachments/assets/dd86b1f9-f3a4-41cb-8cc4-698825e590e2)
 This better?

Reply:
Yeah perfect